.gitignore

```diff
@@ -1,14 +1,17 @@
+# Temporary cache
+.cache/
+
+# Node.js modules
 node_modules/
-npm-debug.log
-.DS_Store
-preview/
-tmp/
-axeReports/
+
+# Test coverage
 coverage/
-public/
-package/package-lock.json
-examples/**/package-lock.json
-*.zip
```

Comment:
**issue** I think that one protects us from commiting the built archive we upload to Github releases. So we'd want to keep it.

Reply:
Ah don't worry, I'll add it back

Mind if I make it self-documenting?

```.gitignore
release-*.zip
```